src/main/java/homework_2/homework2_1/TrafficLight.java

```diff
@@ -0,0 +1,74 @@
+package homework_2.homework2_1;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class TrafficLight {
+    public static final String ANSI_RESET = "\u001B[0m";
+    public static final String ANSI_RED = "\u001B[31m";
+    public static final String ANSI_GREEN = "\u001B[32m";
+    public static final String ANSI_YELLOW = "\u001B[33m";
+
+    public static void main(String[] args) {
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
+            while(true) {
+                System.out.print("Choose the mode:\n1 - seconds\n2 - hh:mm:ss\n0 - exit\n-> ");
+                int mode = Integer.parseInt(reader.readLine());
+                if (mode == 0) {
+                    break;
+                }
+                int seconds = getSeconds(reader, mode);
+
+                printColor(seconds);
+            }
+        } catch (IOException e) {
```

Comment:
Opt: exceptions break loop, is it expected?

Reply:
Yes. Change this to a simple error output?